src/main/java/com/commercetools/sync/services/impl/ProductTypeServiceImpl.java

```diff
@@ -61,8 +63,10 @@ public ProductTypeServiceImpl(@Nonnull final BaseSyncOptions syncOptions) {
                     if (StringUtils.isNotBlank(fetchedTypeKey)) {
                         keyToIdCache.put(fetchedTypeKey, id);
                     } else {
-                        syncOptions.applyWarningCallback(format("ProductType with id: '%s' has no key set. Keys are"
-                                + " required for productType matching.", id));
+                        // syncOptions is ProductSyncOptions
```

Comment:
This comment doesn't seem to be valid. I had a look at an integration test which uses this class and SyncOptionc there is of type `ProductTypeSyncOptions`